### PR TITLE
[bitnami/etcd] Support for customizing standard labels

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR follows up #18154 ensuring this chart is adapted to use new helpers' format, hence adding support for customizing standard labels.

### Benefits

User can customize standard labels.

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

#18154 must be merged **before** this PR

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/etcd/templates/networkpolicy.yaml [hunk 1]
```diff
@@ -56,10 +51,9 @@ spec:
             matchLabels:
               {{ template "common.names.fullname" . }}-client: "true"
         - podSelector:
-            matchLabels: {{- include "common.labels.standard" . | nindent 14 }}
-              {{- if .Values.podLabels }}
-              {{- include "common.tplvalues.render" (dict "value" .Values.podLabels "context" $) | nindent 14 }}
-              {{- end }}
+        - podSelector:
```
Comment (fmulero) [inline bitnami/etcd/templates/networkpolicy.yaml:54]: I think you should remove this line
```suggestion
```

**Review by fmulero — CHANGES_REQUESTED** [review]
Hi @juan131 Thanks a lot for this contribution. There is a typo here, could you check it?

File: bitnami/etcd/templates/networkpolicy.yaml (hunk 1 above)
Comment (juan131) [reply bitnami/etcd/templates/networkpolicy.yaml]: done at https://github.com/bitnami/charts/pull/18298/commits/44f2a7821a601f01f0db5381b60f3485771faaf6

**Review by fmulero — APPROVED** [review]
LGTM

**Pull request merged**